crates/bevy_core_pipeline/src/clear_color.rs

```diff
@@ -29,6 +29,7 @@ pub struct ClearColor(pub Color);
 
 impl Default for ClearColor {
     fn default() -> Self {
-        Self(Color::rgb(0.4, 0.4, 0.4))
+        // Match the bevy website code block color by default.
```

Comment:
Can you please move this to a trait impl doc comment? And maybe describe the color too.

Reply:
You can also add some html to illustrate the color:

```
/// Match the bevy website code block color by default.
///
/// <div style="background: rgb(43, 44, 47); padding: 1em; margin: 1em; color: #fff; max-width: 30em;">
/// This is white text on the default clear color
/// </div>
```

which looks like this (rustdoc dark mode):

<img width="622" alt="image" src="https://github.com/bevyengine/bevy/assets/20063/bac76584-9d86-4265-a96f-47b398d0f0ff">